gflownet/utils/buffer.py

```diff
@@ -259,11 +259,47 @@ def make_data_set(self, config):
         )
         return df, {"x": samples, "energy": energies}
 
-    def compute_stats(self, data):
+    @staticmethod
+    def compute_stats(data):
```

Comment:
Same (re: typehints and docstrings)

Reply:
I will not bother with this one because it may change soon